packages/devtools_app/lib/src/extension_points/extensions_external.dart

```diff
@@ -19,8 +21,10 @@ class ExternalDevToolsExtensionPoints implements DevToolsExtensionPoints {
   Link issueTrackerLink({String? issueDetails}) {
     return Link(
       display: _newDevToolsIssueUriDisplay,
-      url: _newDevToolsGitHubIssueUriLengthSafe(issueDetails: issueDetails)
-          .toString(),
+      url: newDevToolsGitHubIssueUriLengthSafe(
+        additionalInfo: issueDetails,
+        environment: issueLinkDetails(),
```

Comment:
we don't need to pass this in as a parameter. The issueLinkDetails can be looked up directly from inside the `newDevToolsGitHubIssueUriLengthSafe` method

Reply:
It is not very important which method makes this call.
I moved the call for testability: I do not want the test to worry about mocking environment.